composer.json

```diff
@@ -26,12 +26,22 @@
     ],
     "require": {
         "php": "^7.4 || ^8",
+        "cweagans/composer-patches": "^1.7",
         "drupal/core": "^9.2"
     },
     "require-dev": {
         "drupal/coder": "^8.2",
         "phpunit/phpunit": "^7 || ^8",
         "slevomat/coding-standard": "~6.0"
     },
+     "extra": {
```

Comment:
Looks like https://www.drupal.org/project/coder/issues/3250986 has been fixed by @klausi. Perhaps @klausi or @pfrenssen could create a new release for coder?

Reply:
Until then, I think, we can live with the patch